### PR TITLE
RavenDB-15814 When node is passive and license is not activated, stat…

### PR DESCRIPTION
**Pull request by Danielle9897**
…e so on the License View

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/shell/about.ts [hunk 1]
```diff
@@ -8,11 +8,16 @@ import accessManager = require("common/shell/accessManager");
...
 import getLicenseConfigurationSettingsCommand = require("commands/licensing/getLicenseConfigurationSettingsCommand");
+import clusterTopologyManager = require("common/shell/clusterTopologyManager");
+import appUrl = require("common/appUrl");
 
 class about extends viewModelBase {
 
     accessManager = accessManager.default.aboutView;
     
+    clusterViewUrl = appUrl.forCluster();
+    passiveNode = ko.pureComputed(() => clusterTopologyManager.default.nodesCount() === 0);
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/shell/about.ts]: isn't better to use: https://github.com/ravendb/ravendb/blob/135743f79f7a52bfb0f747c432689039f8ed4f15/src/Raven.Studio/typescript/models/database/cluster/clusterTopology.ts#L10 
from 
https://github.com/ravendb/ravendb/blob/v5.0/src/Raven.Studio/typescript/common/shell/clusterTopologyManager.ts#L13

File: src/Raven.Studio/wwwroot/App/views/shell/about.html [hunk 2]
```diff
@@ -55,6 +55,22 @@
...
         </div>
+        <div data-bind="visible: passiveNode">
+            <div class="bg-info text-info flex-horizontal padding">
+                <div class="flex-start"><i class="icon-info"></i></div>
+                <div>The running server is in a <strong>Passive State</strong>, it is not part of a cluster yet.<br>
+                     In order to work, it is required that the server is part of a cluster.<br><br>
+                    Either one of the following can be done to <strong>Bootstrap a Cluster</strong>:<br>
+                     <ul>
+                         <li>Create a new database</li>
+                         <li>Register a license (if not registered yet)</li>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/shell/about.html]: do we know if license is registered or no?

File: src/Raven.Studio/wwwroot/App/views/shell/about.html [hunk 3]
```diff
@@ -55,6 +55,22 @@
                 </div>
             </div>
         </div>
+        <div data-bind="visible: passiveNode">
+            <div class="bg-info text-info flex-horizontal padding">
+                <div class="flex-start"><i class="icon-info"></i></div>
+                <div>The running server is in a <strong>Passive State</strong>, it is not part of a cluster yet.<br>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/shell/about.html]: I'd join last sentence with this one.

File: src/Raven.Studio/wwwroot/App/views/shell/about.html (hunk 2 above)
Comment (ml054) [reply src/Raven.Studio/wwwroot/App/views/shell/about.html]: "In order to work" - what's not working?

File: src/Raven.Studio/typescript/viewmodels/shell/about.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/shell/about.ts]: done

File: src/Raven.Studio/wwwroot/App/views/shell/about.html (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/shell/about.html]: Refactored text to be:

image

File: src/Raven.Studio/wwwroot/App/views/shell/about.html (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/shell/about.html]: > do we know if license is registered or no?

As far as I see, at this point, we don't have this info
The '`registered`' observable https://github.com/ravendb/ravendb/blob/efba92603d4ba571907188e038c03b0b91250e13/src/Raven.Studio/typescript/viewmodels/shell/about.ts#L189
returns relevant info **only** when we have a cluster

**Review by ml054 — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**